recipe/meta.yaml

```diff
@@ -46,7 +56,6 @@ about:
     uvloop is a fast, drop-in replacement of the built-in asyncio event loop.
     uvloop is implemented in Cython and uses libuv under the hood.
   doc_url: http://uvloop.readthedocs.org
-  dev_url: https://github.com/MagicStack/uvloop
```

Comment:
Know this is redundant, but let's leave it. Some folks like having this stuff show up in the README and Anaconda.org

Reply:
94a1f7e